tests/services/schemas/test_additional_description.py

```diff
@@ -89,6 +99,7 @@ def test_invalid_description_type(vocabulary_clear):
     )
 
 
+@pytest.mark.skip(reason="currently don't know how to test this")
```

Comment:
I will do another pass and test this: https://github.com/inveniosoftware/invenio-rdm-records/issues/624